src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -208,6 +208,17 @@ protected Object decode(
                                 case 3:
                                     position.setDeviceTime(new Date(buf.readUnsignedIntLE() * 1000));
                                     break;
+                                case 4:
+                                    value = buf.readUnsignedByte();
+                                    position.set(Position.KEY_ALARM, BitUtil.check(value, 2));
+                                    int guardMode = BitUtil.between(value, 3, 4);
+                                    position.set(Position.KEY_ARMED, (0 < guardMode) && (guardMode < 3));
+                                    break;
+
+                                case 5:
+                                    value = buf.readUnsignedByte();
+                                    position.set(Position.KEY_ROAMING, BitUtil.check(value, 6));
```

Comment:
This should be set only if true. We don't want every message to include this flag.

Reply:
Ok, fix it